src/components/insight-box/insight-box.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import clsx from "clsx";
+
+export interface InsightProps {
+  title: string;
+  message: string;
+  icon?: React.ReactNode;
```

Comment:
This works but it doesn't seem very type-safe considering you can pass _any_ component into it...
The Button component for example has an icon attribute that is of type `IconDefinition` which seems to be used by the FontAwesome icons for type-safety - I wonder if you can make this icon more type-safe or find a way to use the lightbulb from FontAwesome and stylize it?

Reply:
@Aivaxela I saw this component as a mirrored carbon copy of the notification box. 
I copied the notification box code and adjusted it accordingly. 
This implementation of the icon is from there.